packages/cli-kit/src/public/node/base-command.ts

```diff
@@ -104,12 +104,16 @@ abstract class BaseCommand extends Command {
     return result
   }
 
-  protected async environmentsPath(rawFlags: {path?: string}): Promise<string> {
-    return rawFlags.path || cwd()
+  protected async environmentsFilePath(rawFlags: {path?: string}): Promise<string | undefined> {
+    const basePath = rawFlags.path || cwd()
+    return findPathUp(this.environmentsFilename(), {
+      cwd: basePath,
+      type: 'file',
+    })
   }
 
-  protected findUpForEnvironments(): boolean {
-    return true
+  protected environmentsFilename(): string {
```

Comment:
is it mandatory to implement this? If not, should we change the return type to `string | undefine` and remove the `throw`?